### PR TITLE
Fixed search box

### PR DESCRIPTION
**Pull request by timschoch**
This fixes the search box to the top of the browser window. By this you can easely seach on if the first element was not the one you where looking for.
First Commit is to fix the search bar, second one prevents the item titles from hiding behind the search bar

### CODE REVIEW

File: scss/partials/_header.scss [hunk 1]
```diff
@@ -3,11 +3,20 @@
  */
 .header {
   @include clearfix;
+  @include fixed( $left: $sidebar-width, $top: 0, $right: 0 );
```
Comment (HugoGiraudel) [inline scss/partials/_header.scss]: Any chance you could remove the space after `(` and the space before `)` perhaps? :)

**Comment by HugoGiraudel** [comment]
I'll test this in the next couple of days then merge it. Thanks! :)

File: scss/partials/_header.scss (hunk 1 above)
Comment (timschoch) [inline scss/partials/_header.scss]: Done :)

**Comment by HugoGiraudel** [comment]
Thanks!

**Pull request merged**